### PR TITLE
Docs 2641 publish scale network page video

### PR DESCRIPTION
**Pull request by micjohnson777**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/Network/_index.md [hunk 1]
```diff
@@ -181,119 +186,119 @@ To configure OpenVPN (server or client) on TrueNAS, select the networking creden
...
+  **TAP** devices always use the **SUBNET** for **Topology**.
 
-TrueNAS automatically applies the *Topology* selection to any connected clients.
+TrueNAS automatically applies the **Topology** selection to any connected clients.
 
 #### TLS Crypt Auth
 
-When users enable *TLS Crypt Auth Enabled*, TrueNAS generates a static key for the *TLS Crypt Auth* field after saving the options.
-To change this key, click *Renew Static Key*.
+When users enable **TLS Crypt Auth Enabled**, TrueNAS generates a static key for the **TLS Crypt Aut*h* field after saving the options.
```
Comment (Mrt134) [inline content/SCALE/Network/_index.md:249]: Typo fix
```suggestion
When users enable **TLS Crypt Auth Enabled**, TrueNAS generates a static key for the **TLS Crypt Auth** field after saving the options.
```

File: content/SCALE/Network/_index.md [hunk 2]
```diff
@@ -181,119 +186,119 @@ To configure OpenVPN (server or client) on TrueNAS, select the networking creden
...
 
-When finished configuring the Server or Client service, click *Save*.
-Start the service by clicking the play button next to it in the OpenVPN window. 
+When finished configuring the server or client service, click **Save**.
+Start the service by clicking the play button next to it in the **OpenVPN** window. 
 
-You may also start the service by going to **System Settings > Services** and clicking the *State* toggle.
-Setting *Start Automatically* starts the service when TrueNAS completes booting and runs the network and data pools.
+You may also start the service by going to **System Settings > Services** and clicking the **State** toggle.
+Setting *S*tart Automatically** starts the service when TrueNAS completes booting and runs the network and data pools.
```
Comment (Mrt134) [inline content/SCALE/Network/_index.md:295]:
```suggestion
Setting **Start Automatically** starts the service when TrueNAS completes booting and runs the network and data pools.
```

**Review by Mrt134 — COMMENTED** [review]
Found a couple typos.

**Review by BonnieF — CHANGES_REQUESTED** [review]
The video for /scale/Network/ "Networking Tour Video" expand area and the one under the Global configuration are playing the same video - even though they are named differently. Can you add the correct video to whichever one is wrong?

**Comment by tonyriv3** [comment]
@micjohnson777 
The two videos are identical. The `scaleangelfishnetworking` is the most up-to-date version, so you should only use that one.

**Review by Mrt134 — APPROVED** [review]

**Comment by Mrt134** [comment]
Link check hit one timeout issue with MIT PGP public key server. Did more digging and the key lookup function on the site is slow enough that the checker isn't likely to pass it. We'll either need to replace this URL with a link to a site that is more performant, or add an exception to the checker and run the risk of missing early warnings if something permanently changes about the link location.

**Pull request merged**